conf/cortx_rgw.conf

```diff
@@ -11,15 +11,21 @@
     rgw thread pool size = 10
     rgw max concurrent request = 10
     rgw init timeout = 300
-    rgw gc max objs = 32
-    rgw gc obj min wait = 1800
-    rgw gc processor max time = 3600
-    rgw gc processor period = 3600
     rgw data path = /var/lib/ceph/radosgw/<cluster-id>   # Cluster id of cortx cluster
 
     # Enabling cache causes data retrieval inconsistencies for get-object, head-object etc. ref <CORTX-31109>
     rgw cache enabled = false
 
+    # GC configuration paraneters
+    rgw enable gc threads = true
```

Comment:
@sachinpunadikar should we keep this value as '**false**' since GC work is not completed yet

Reply:
We dont have backend code yet in `main` so even if this is set to true, it will not have any effect.

Reply:
@sachitanands has kept it false for now.

Reply:
Lets keep it true , we want this as the default value for RGW+Motr 
Current RGW code has no impact of this value. Once we merge GC related code into main, then above value will help in starting GC framework by default.

Reply:
Ok, we just wanted to avoid any unexpected errors in deployments if some incremental changes in RGW GC flow is merged and this flag may end up enabling that code flow. 
Anyways will keep it true, @sachitanands please do the needful

Reply:
thanks @cdeshmukh .